### DISCUSSION TITLE
Welcome to mgob Discussions!

### DISCUSSION THREAD

**Discussion by maxisam** [maxisam/mgob#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheDukeDK** [comment]
Hello,

Hope "Discussions" is still the place to interact. We are onboarding Community MongoDB Operator to our K8s clusters and were looking for a backup/restore solution and found you guys. :smile: 

First off. We really appreciate you folks for maintaining and providing this!

Secondly, we just wanted to ask if new releases are supported with the Helm chart and it is still being maintained? We just noticed that the appVersion hadn't been updated in the chart for a while and thought we would ask.

**Reply by maxisam** [reply]
yeah, sorry, my day job is a bit busy right now. Working a lot of hours everyday for a month or two. 

here is the new chart https://github.com/maxisam/mgob/releases/tag/mgob-2.1.1
